thanos/charts/templates/ruler/alertmanager-config.yaml

```diff
@@ -0,0 +1,11 @@
+{{ if .Values.thanos.ruler.enabled }}
```

Comment:
maybe we can make this config optional. Like `.Values.thanos.ruler.amconfig.enabled`

It defaults to false and expects the AM to be present in the cluster. Should have a not in the `values.yaml` to point out that this needs to be specified, if there is no AM around.